network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -1343,10 +1371,11 @@ def get_context(self, request, *args, **kwargs):
                 self.cutoff_date,
                 authenticated,
                 key,
-                ProductPage.objects.all()
+                ProductPage.objects.exclude(product_categories__in=self.excluded_categories.all().values_list('id'))
             )
 
-        context['categories'] = BuyersGuideProductCategory.objects.filter(hidden=False)
+        context['categories'] = BuyersGuideProductCategory.objects.exclude(
+            excludedcategories__in=self.excluded_categories.all())
```

Comment:
We still want to show these categories in the nav, we just don't want to show products from those categories in the main listing.